packages/matchbox/src/components/Modal/styles.js

```diff
@@ -10,6 +10,7 @@ export const base = props => {
     left: 0;
     width: 100vw;
     height: 100vh;
+    padding: ${tokens.spacing_700} 0;
```

Comment:
Do we want this to be configurable?

Reply:
Probably not – since this is more of a layout component. Similar to page, its probably better to be restrictive